src/components/Button/README.md

```diff
@@ -254,6 +254,31 @@ Use when a button has been pressed and the associated action is in progress.
 <Button loading>Save product</Button>
 ```
 
+<!-- content-for: web -->
```

Comment:
Example section summarizing accessibility-related information for the component, including the recommended use of props and expected interactions.